docs/intro/quickstart/modeling.rst

```diff
@@ -0,0 +1,51 @@
+.. _ref_quickstart_modeling:
```

Comment:
What if we add a bit more here? Since it's relatively short, maybe we can also explain links here?

Reply:
Yeah, I think in the mock-data -> Gel data version of this modeling section, we can look at the structure of the mock data and talk more about how to map this structure onto the Gel schema language's data modeling concepts, like links, cardinality, sets vs arrays, etc.